### ISSUE TITLE
Discord servers with emotes in the name don't show up on the server list

### ISSUE THREAD

**Issue by firedo** [designeerlabs/discord-vr#2]
Discord servers with emotes (at least fire emotes) in the name don't show up on the server list. It seems to be shown as a empty line and if I select it the channel list says "No results found".

![DiscordVR-empty](https://user-images.githubusercontent.com/27922656/118504511-3f1d4f80-b734-11eb-8a7d-0fc45f6dc318.png)

**Comment by designeerlabs** [comment]
Yep, that's because I can't figure out how to display non BMP characters. If you know a way of doing it lmk
